src/tools/pub/pubspec.md

```diff
@@ -172,22 +167,22 @@ The description is plain text: no markdown or HTML.
 
 ### Author/authors
 
-You're encouraged to use these fields to describe the author(s) of your package
-and provide contact information. Use `author` if your package has a
-single author, or use `authors` with a YAML list if more than one
-person wrote the package. Each author can be either a single name
+_Deprecated._ Use a [verified publisher][] instead.
+
+[verified publisher]: /tools/pub/verified-publishers
+
+You might see an `author` or `authors` section in old pubspecs.
+These optional fields were a way to describe
+the author(s) of your package and to provide contact information.
+Each author could be either a single name
 (`Natalie Weizenbaum`) or a name and an email address
-(`Natalie Weizenbaum <nweiz@google.com>`). For example:
+(`Natalie Weizenbaum <nweiz@google.com>`).
+However, these values weren't verified,
+and pub.dev no longer displays them.
 
-{% prettify yaml %}
-authors:
-- Natalie Weizenbaum <nweiz@google.com>
-- Bob Nystrom <rnystrom@google.com>
-{% endprettify %}
+As of Dart 2.7, the `pub publish` command
```

Comment:
maybe add here: ", and the pub.dev site no longer displays the `author(s)` on a package page.".

Reply:
I say that up above.

Reply:
Sorry, missed that. I think it would be easier to not miss if we did something like this:

```
You might see an `author` or `authors` section in old pubspecs.
These optional fields were a way to describe
the author(s) of your package and to provide contact information.
Each author could be either a single name
(`Natalie Weizenbaum`) or a name and an email address
(`Natalie Weizenbaum <nweiz@google.com>`).

As of Dart 2.7, pub.dev no longer displays them and the `pub publish` command
displays a warning if your pubspec has an `author` or `authors` section.
```

Reply:
The thing is, pub.dev no longer displaying the fields is independent of 2.7. But I'll try to reword it to remove that dependency.